qadence/operations.py

```diff
@@ -387,7 +369,7 @@ def __pow__(self, power: int) -> AbstractBlock:
         return self
 
     def dagger(self) -> Zero:
-        return Zero()
+        return self
```

Comment:
Do you need to override the method from the `PrimitiveBlock` here?

Reply:
you mean putting the override decorator?

Reply:
yes

Reply:
done